xbmc/video/VideoDatabase.cpp

```diff
@@ -2072,8 +2072,6 @@ std::string CVideoDatabase::GetValueString(const CVideoInfoTag &details, int min
   std::vector<std::string> conditions;
   for (int i = min + 1; i < max; ++i)
   {
-    if (offsets[i].type == NULL || offsets[i].offset == NULL)
```

Comment:
Before the compiler replaced nulls with 0 so if it was null (or 0 after the compiling) it will hit "continue" and pass it to the next field. 
But now that's a valid value for switch a specific check is pointless.. now it will enter in the switch and since there's not the unused type it won't find a match and it will continue automatically to the next one